forum/qa-plugin/user-changeable-theme/user-theme-layer.php

```diff
@@ -38,58 +38,54 @@ public function head_lines()
                 $this->save_theme($_POST['select_theme']);
             }
             $this->content['head_lines'][] = '<style>
-				.widget-select-theme {
-					display: -webkit-flex;
-					display: -ms-flexbox;
-					display: flex;
-					-webkit-justify-content: center;
-					-ms-flex-pack: center;
-					        justify-content: center;
-					-webkit-align-items: center;
-					-ms-flex-align: center;
-					        align-items: center
-				}
+              .widget-select-theme {
+                display: -webkit-flex;
+                display: -ms-flexbox;
+                display: flex;
+                -webkit-justify-content: center;
+                -ms-flex-pack: center;
+                        justify-content: center;
+                -webkit-align-items: center;
+                -ms-flex-align: center;
+                        align-items: center
+              }
 
-				.widget-select-theme__text {
-					margin: 0 10px 2px 0;
-				}
+              .widget-select-theme__text {
+                margin: 0 10px 2px 0;
+              }
 
-				.widget-select-theme__button {
-					background: none;
-					color: #34495e;
-					background: white;
-					border: 1px solid #34495e;
-					width: 80px;
-					padding: 10px 0;
-					transition: all 0.2s;
-					outline: none;
-				}
+              .widget-select-theme__button {
+                background: none;
+                color: #34495e;
+                background: white;
+                border: 1px solid #34495e;
+                width: 80px;
+                padding: 10px 0;
+                transition: all 0.2s;
+                outline: none;
+              }
 
-				.widget-select-theme__button--dark {
-					background: #34495e;
-					color: white;
-					border-left: none;
-				}
+              .widget-select-theme__button--dark {
+                background: #34495e;
+                color: white;
+                border-left: none;
+              }
 
-				.widget-select-theme__button--dark:hover, .widget-select-theme__button--dark:focus {
-					background: #1e2f40;
-				}
+              .widget-select-theme__button--dark:hover, .widget-select-theme__button--dark:focus {
+                background: #1e2f40;
+              }
 
-				.widget-select-theme__button--light {
-					pointer-events: none;
-				}
+              .widget-select-theme__button--light {
+                pointer-events: none;
+              }
 
-				/* force to display theme buttons (mobile) */
-				@media (max-width: 1179px) {
-					.qa-sidepanel {
-						width: 100% !important;
-						display: block !important;
-					}
-
-					.qa-sidepanel > div:not(.qa-widgets-side-bottom) {
-						display: none;
-					}
-				}
+              /* force to display theme buttons (mobile) */
+              @media (max-width: 1179px) {
+                .qa-sidepanel {
+                  width: 100% !important;
+                  display: block !important;
+                }
+              }
```

Comment:
Tak jak mówiłem wcześniej @Argeento, w wersji komputerowej pojawia się brzydkie rozciągnięte menu na dole strony, gdy jest ona zwężona. Oczywiście to nie jest bardzo znacząca wada, prawie nikt na to i tak nie spojrzy, no ale wg mnie lepiej nie eksponować takich niedoróbek. Na pewien czas (zanim się nie dorobimy prawdziwego RWD) można dodać warunek z funkcją qa_is_mobile_probably() i wg niego dodawać to display: none. Za niedługo spróbuję wrzucić commita to poprawiającego.